openstack/clustering/v1/clusters/requests.go

```diff
@@ -0,0 +1,48 @@
+package clusters
+
+import (
+	"net/http"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+// CreateOptsBuilder Builder.
+type CreateOptsBuilder interface {
+	ToClusterCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts params
+type CreateOpts struct {
+	Name            string                 `json:"name" required:"true"`
+	DesiredCapacity int                    `json:"desired_capacity" required:"false"`
+	ProfileID       string                 `json:"profile_id" required:"true"`
+	MinSize         int                    `json:"min_size,omitempty"`
```

Comment:
Is `0` a valid value to pass? If so, let's change this to `*int`. 

`0` is `int`'s zero-value and combined with `omitempty` means it'll never be sent in the request.

Reply:
Good catch